SIL.Windows.Forms/Miscellaneous/SILAboutBox.cs

```diff
@@ -146,7 +156,7 @@ private string GetBuiltOnDate()
 
 			// Use UTC for calculation of build-on-date so that we get the same date regardless
 			// of timezone setting.
-			return string.Format("Built on {0}", fi.CreationTimeUtc.ToString("dd-MMM-yyyy"));
+			return string.Format(LocalizationManager.GetString("AboutDialog.BuiltOnDate", "Built on {0}"), fi.CreationTimeUtc.ToString("dd-MMM-yyyy"));
```

Comment:
This could really use a translator note as the 3rd parameter (wasn't there before either, but now that we're looking at it).